### DISCUSSION TITLE
Why is it not possible to set NS records manually?

### DISCUSSION THREAD

**Discussion by arimendelow** [vercel/vercel#4342] [General] [answered]
Hi,

I need to set NS DNS records for a domain managed with Now. It's crazy that this isn't one of the supported self-managed DNS records.

Why is this?

See here for more information on why this is something a user would want to do: https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/MigratingSubdomain.html

Thanks!

**Comment by mcsdevv** [comment, marked as answer]
Hey @arimendelow, thanks for raising this. The ability to set NS on the customer side was never implemented originally but is much requested. We are just about to release some big DNS improvements and in the near future, this will allow for the ability to set NS also.

**Comment by bigsee** [comment]
I have the same issue needing to migrate a subdomain to Route 53 of a domain I bought through Now/Vercel. Is there any way to update these records in the meantime? If not, any rough timescale where in the roadmap this is?
